scripts/flying-tokens.js

```diff
@@ -69,23 +114,24 @@ export async function fly(token, elevation) {
         await flyZoom(token, elevation)
         await flyingFX(token, elevation)
         if (notificationOutput)
-            ui.notifications.info(token.actor.name + ' is flying at <b>' + elevation + ' feet</b> high.')
+            ui.notifications.info(token.name + ' is flying at <b>' + elevation + ' feet</b> high.')
         if (chatOutput)
-            await chatMessage(`<img src='${token.actor.img}' width='32' style='border:none'> ${token.actor.name} is flying at <b>${elevation} feet</b> high.`)
+            await chatMessage(`<img src='${token.texture.src}' width='32' style='border:none'> ${token.name} is flying at <b>${elevation} feet</b> high.`)
```

Comment:
replaced for `token.name` because if this is used for NPCs with the same actor name and different tokens names, so avoid confusion

Reply:
lol I sounded like an insane above. What I mean is that this is meant to avoid confusion for multiple unlinked actors.